operating/view-dashboard.adoc

```diff
@@ -0,0 +1,63 @@
+[id="view-dashboard"]
+= Viewing the dashboard
+include::modules/common-attributes.adoc[]
+:context: view-dashboard
+
+toc::[]
+
+[role="_abstract"]
+{product-title} dashboard enables you to visually track and analyze key metrics such as risk across your environment, compliance status, and policy violations.
```

Comment:
I think this should start with "The" since you have "dashboard" after the product name.